src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.ts

```diff
@@ -32,55 +44,42 @@ export class HearingActualSummaryComponent implements OnInit {
     this.setPartyData();
   }
 
-  public setPartyData() {
+  public setPartyData(): void {
     const plannedParties = this.hearingActualsMainModel.hearingPlanned.plannedHearingDays[0].parties;
     const actualParties = this.actualHearingDays.actualDayParties;
     actualParties.forEach((actualPartyInfo) => {
       const plannedPartyInfo = plannedParties.find((plannedParty) => plannedParty.partyID === actualPartyInfo.actualPartyId);
+      // if that is in planned party that will be put in participants
       if (plannedPartyInfo) {
         const partyDetail: ActualDayPartyModel = {
           individualDetails: {
-            firstName: plannedPartyInfo.individualDetails.firstName,
-            lastName: plannedPartyInfo.individualDetails.lastName,
+            firstName: actualPartyInfo.individualDetails.firstName,
+            lastName: actualPartyInfo.individualDetails.lastName,
           },
-          actualOrganisationName: plannedPartyInfo.organisationDetails ? plannedPartyInfo.organisationDetails.name : '',
+          actualOrganisationName: actualPartyInfo.actualOrganisationName ? actualPartyInfo.actualOrganisationName : '',
```

Comment:
not sure if this works, actualPartyInfo.actualOrganisationName || ''

Reply:
good one this is more elegant.